### PR TITLE
Add "get_fixture_content" method to the base pack resource test classes

### PR DESCRIPTION
**Pull request by Kami**
This PR adds "get_fixture_content" method to the base pack resource test classes. This method enforces fixture file location conventions and allows user to load raw fixture content from a fixture file on disk.

We discussed this on a st2contrib PR recently - the functionality is very simple, but we still think there is value in it since it enforces fixture file location convention (`<pack path>/tests/fixtures/`).

### CODE REVIEW

**Comment by jjm** [comment]
This will be really useful, I'm finishing up a PR for st2contrib which I'm happy to up date to use these helper functions prior to it being merged. 

Thanks @Kami!

**Comment by lakshmi-kannan** [comment]
👍 , I like the convention as well, personally.

File: st2tests/st2tests/pack_resource.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+class BasePackResourceTestCase(TestCase):
+    """
+    Base test class for all the pack resource test classes.
+
+    Contains some utility methods for loading fixtures from disk, etc.
+    """
+
+    def get_fixture_content(self, fixture_path):
```
Comment (lakshmi-kannan) [inline st2tests/st2tests/pack_resource.py]: I don't think we should allow nesting. Better to support fixture_file_name. Wdyt?

Comment (Kami) [inline st2tests/st2tests/pack_resource.py]: I think it's fine if user wants to store fixture in a sub-directory (e.g. `tests/fixtures/api_responses/my_fixture.json` or similar)

**Pull request merged**